assets/migrations/scripts/20200629122125_add_columns_in_event_metadata_table.sql

```diff
@@ -0,0 +1,14 @@
+-- // add columns in event metadata table
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE core.event_metadata ADD COLUMN plan_identifier VARCHAR DEFAULT NULL;
+
+CREATE INDEX event_metadata_plan_identifier_index ON core.event_metadata(plan_identifier);
```

Comment:
Add a query to populate existing events with the plan Identifier
see related issue https://github.com/OpenSRP/opensrp-server-configs/pull/80#discussion_r449421612

Reply:
Done